### DISCUSSION TITLE
Outbound IP address

### DISCUSSION THREAD

**Discussion by bonald** [saucelabs/forwarder#730] [General]
Is there a way to select an outbound IP ? Looking to implement round-robin or similar per request.

**Comment by mmatczuk** [comment]
Not at the moment but it's fairly easy to add.

To implement it you need to add the `LocalIP` it to DialConfig

https://github.com/saucelabs/forwarder/blob/6ad763839eb9564879ebb96b01b85713c753168e/net.go#L23

and set `LocalAddress` in Dialer 

https://github.com/saucelabs/forwarder/blob/6ad763839eb9564879ebb96b01b85713c753168e/net.go#L52
